File: clients/android/app/src/main/java/com/yeplist/app/data/remote/dto/TodoListDto.kt
package com.yeplist.app.data.remote.dto

import com.yeplist.app.data.local.entity.TodoListEntity

data class TodoListDto(
    val listId: Long,
    val name: String,
    val sortOrder: Int,
    val createdDate: String,
    val modifiedDate: String
) {
    fun toEntity(): TodoListEntity = TodoListEntity(
        listId = listId,
        name = name,
        sortOrder = sortOrder,
        createdDate = createdDate,
        modifiedDate = modifiedDate
    )
}

File: backend/src/ToDoList.Core/Dtos/TodoListDto.cs
using System;

namespace ToDoList.Core.Dtos
{
    public class TodoListDto
    {
        public long ListId { get; set; }
        public string Name { get; set; } = string.Empty;
        public int SortOrder { get; set; }
        public int Version { get; set; }
        public DateTime CreatedDate { get; set; }
        public DateTime ModifiedDate { get; set; }
        public DateTime ClientModifiedDate { get; set; }
    }
}
